server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateSnapshotPlan.java

```diff
@@ -35,4 +35,9 @@ public CreateSnapshotPlan() {
   public List<Path> getPaths() {
     return new ArrayList<>();
   }
+
+  @Override
+  public List<String> getPathsStrings() {
+    return new ArrayList<>();
```

Comment:
Use Collections.emptyList().

Reply:
I'll change it. Thanks for your suggestion.